package.json

```diff
@@ -89,14 +90,15 @@
     ],
     "moduleNameMapper": {
       "^.+\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/config/jest/FileStub.js",
-      "^.+\\.css$": "<rootDir>/config/jest/CSSStub.js"
+      "^.+\\.css$": "<rootDir>/config/jest/CSSStub.js",
+      "ovirtapi": "<rootDir>/src/mock/ovirtapi.mock.js"
     },
     "scriptPreprocessor": "<rootDir>/config/jest/transform.js",
     "setupFiles": [
       "<rootDir>/config/polyfills.js"
     ],
     "testPathIgnorePatterns": [
-      "<rootDir>/(build|docs|node_modules)/"
+      "<rootDir>/(build|docs|node_modules|__tests__\/helper|config)/"
     ],
     "testEnvironment": "node",
     "testRegex": "(/__tests__/.*|\\.(test|spec))\\.(js|jsx)$"
```

Comment:
In one case it better to use \_\_tests\_\_, because it easy to find in directory list, but I can't say 100% that this directory is so important that we must distinguish it.